app/services/jbpm_process_service.rb

```diff
@@ -33,12 +33,27 @@ def process_options
       options = {
         'request'         => request,
         'request_context' => request.request_context.as_json,
-        'groups'          => [group].as_json
+        'groups'          => enhance_groups([group].as_json)
       }
     end
     options
   end
 
+  def enhance_groups(groups_json)
+    random_access_keys = []
+    groups_json.each do |group_json|
+      group_json['users']&.each do |user_json|
+        access_key = SecureRandom.hex(16)
```

Comment:
If we ever want to extend this value to a longer value all previously created access keys would still continue to work, correct?

Reply:
yes